src/components/AuthForm.js

```diff
@@ -20,10 +20,24 @@ function AuthForm (props) {
       }
     }
 
-    const urlForUsers = endpointApiUrl.concat("/","users")
+    const urlForUsers = endpointApiUrl.concat("/users")
```

Comment:
Similar to @MjxOro comment, have a look at template strings.

Reply:
All updated. Thanks!